packages/cli/src/ceramic-daemon.ts

```diff
@@ -295,14 +295,15 @@ export class CeramicDaemon {
     const ceramic = new Ceramic(modules, params)
     if (opts.stateStore?.mode == StateStoreMode.S3) {
       const s3Store = new S3Store(opts.stateStore?.s3Bucket, params.networkOptions.name)
-      await ceramic.repository.injectStateStore(s3Store)
+      await ceramic.repository.injectKeyValueStore(s3Store)
     }
     const did = new DID({ resolver: makeResolvers(ceramic, ceramicConfig, opts) })
     ceramic.did = did
     await ceramic._init(true)
 
     const daemon = new CeramicDaemon(ceramic, opts)
     await daemon.listen()
+    ceramic.repository.resumeRunningStatesFromAnchorRequestStore()
```

Comment:
Since it is `async` operation, mind adding an error handler here, please?

Non-actionable: How about handling shutdown gracefully?

Reply:
UPD: Graceful shutdown is here. Have not seen that.